website/content/docs/secrets/pki/considerations.mdx

```diff
@@ -29,7 +50,7 @@ If you plan on using intermediate CAs with Vault, it is suggested that you let
 Vault create CSRs and do not export the private key, then sign those with your
 root CA (which may be a second mount of the `pki` secrets engine).
 
-### Managed Keys
+## Managed Keys
```

Comment:
I liked placing this under the root CA header personally, to show that its most relevant there, but up to you :-) 

There are other nested headers FWIW, that the ToC doesn't reflect (e.g., `Always Configure a Default Issuer` is nested below `One CA Certificate, One Secrets Engine`). 

Might be nice to align everything?

Reply:
Thank you, looks great!